azure-pipelines.yml

```diff
@@ -97,9 +97,9 @@ jobs:
         - wheels_macos
         - sdist
       dependsOn:
-        - py36_test
         - py37_test
+        - py38_test
         - py38_test_online
         - py38_test_32bit
-        - py36_test_oldestdeps
-        - py36_conda
+        - py38_test_oldestdeps
+        - py38_conda
```

Comment:
Is the whole test suite run on `py38_conda`? If so it seems a bit of a waste given there's already a set of tests run on `py38_test`.

Reply:
yeah, it needs to run the whole test suite. We could, but I am not sure I want to swap the conda build to macos to remove the duplication but I would rather keep them separate.